README.md

```diff
@@ -1,16 +1,117 @@
-# Ministry of Justice Template Repository
+# Modernisation Platform Terraform Loadbalancer Module with Access Logs enabled
+[![repo standards badge](https://img.shields.io/badge/dynamic/json?color=blue&style=for-the-badge&logo=github&label=MoJ%20Compliant&query=%24.data%5B%3F%28%40.name%20%3D%3D%20%22modernisation-platform-terraform-s3-bucket%22%29%5D.status&url=https%3A%2F%2Foperations-engineering-reports.cloud-platform.service.justice.gov.uk%2Fgithub_repositories)](https://operations-engineering-reports.cloud-platform.service.justice.gov.uk/github_repositories#modernisation-platform-terraform-s3-bucket "Link to report")
```

Comment:
The 2 references to `modernisation-platform-terraform-s3-bucket` should be updated to `modernisation-platform-terraform-loadbalancer`